source/_docs/installation/run-beta.markdown

```diff
@@ -0,0 +1,11 @@
+---
+title: "Run the beta version on Home Assistant"
+description: "Instructions to run the Home Assistant beta."
+---
+
+If you would like to test next release before anyone else, you can install the beta version released every three weeks:
+
+1. Backup your installation, using the snapshot functionality Home Assistant offers.
+2. Check the [Home Assistant Beta release notes](https://rc.home-assistant.io/latest-release-notes/) for breaking changes. Be sure to check all release notes between the version you are running and the one you are upgrading to. Use the search function in your browser (`CTRL + f`) and search for **Breaking Changes**.
+3. Select _System_ tab from the _Supervisor_ menu, then select _Join Beta Channel_ under _Supervisor_, then select _Reload_.
+4. Select _Dashboard_ tab from the _Supervisor_ menu, and then select _Update_.
```

Comment:
This now all of a sudden is a general installation instruction, which it is not. The documentation written here is specific for Supervisor enabled systems, which covers only 2 of the 4 installation methods we have.